hera_mc/geo_handling.py

```diff
@@ -194,12 +194,16 @@ def find_antenna_at_station(self, station, query_date):
         ctr = 0
         for conn in connected_antenna:
             if conn.stop_gpstime is None or query_date.gps <= conn.stop_gpstime:
+                if ctr:
+                    print("Already connected: {} -> {}".format(antenna_connected.upstream_part, antenna_connected.downstream_part))
+                    print("...trying to add:  {} -> {}".format(conn.upstream_part, conn.downstream_part))
```

Comment:
This confuses me. This method is supposed to "check to see what antenna is at a station". I would not expect to get print statements about trying to add things from this method.

Reply:
Is the top line docstring description accurate? If so, what are these prints supposed to convey to me?

Reply:
Confusion warranted ;-).  I've fixed that.